src/main/kotlin/no/nav/soknad/arkivering/soknadsarkiverer/service/TaskListService.kt

```diff
@@ -87,6 +87,7 @@ class TaskListService(
 			if (processRun && task.isRunningLock.tryAcquire()) {
 				jobMap[key] = GlobalScope.launch { start(key, state) } // Start new thread for handling archiving of application with key
 			}
+			metrics.setTasksGivenUpOn(loggedTaskStates.values.filter{v -> v == EventTypes.FAILURE}.size)
```

Comment:
Jeg ville flyttet det ut til en egen metode, dels siden den finnes på to sted, men framforallt for at gjöre det mer lettlest og minske kognitiv load for den som leser koden

Reply:
Flyttet denne ut linjen i egen metode